t/ui/18-tests-details.t

```diff
@@ -641,6 +641,15 @@ subtest 'additional investigation notes provided on new failed' => sub {
         'investigation status content shown as table');
 };
 
+subtest 'alertbox shown if there first_bad test available' => sub {
```

Comment:
"if there first_bad test available"?

How about "link to 'first bad' shown if not already on 'first bad'"?